docs/victoriatraces/data-ingestion/opentelemetry.md

```diff
@@ -0,0 +1,82 @@
+---
+weight: 4
+title: OpenTelemetry setup
```

Comment:
This page is not present in the left menu. I've tried to fix it but without luck so far

<img width="1512" alt="Screenshot 2025-06-27 at 15 29 55" src="https://github.com/user-attachments/assets/9a8b2ba4-2ce9-42b6-a6a0-64afdc4e3cc0" />

@AndrewChubatiuk maybe you might have some idea why it's not included.